js/app.js

```diff
@@ -1,20 +1,82 @@
 $(document).ready(function() {
-}
+	$('.start').on('click', function(e) {
+		e.preventDefault();
+		$('#intro-container').hide();
+		$('#question-container').show();
+	});
+	$('q-submit-button').on('click', function(e) {
+		e.preventDefault();
 
-// Answer 5 questions
+		quiz.checkAnswer();
+		//Hide question div
+		//Show answer div
+	});
+	$('nextButton').on('click', function(e) {
+		e.preventDefault();
 
-// Show each question one at a time
+		quiz.nextQuestion();
+		// Hide answer div
 
-// Tell user each question's number
+		// Show questions div
+	});
 
-// Show how many questions remain
+});
 
-// Prevent user from skipping questions
 
-// Gives user some way to answer each question
+var questions = [{
+	question: "What is the capital of Australia?",
+	choices: ["Sydney", "Canberra", "Melbourne", "Adelaide"],
+	correct: 0,
+	},
+	{
+	question: "By area, what is the smallest country on the planet?",
+	answers: ["Vatican City", "Monaco", "Nauru", "Madagascar"],
+	correct: 0,
+	},
+	{
+	question: "What is the name of the tallest uninterrupted waterfall in the world?",
+	choices: ["Victoria Falls", "Angel Falls", "Niagara Falls", "Alamere Falls"],
+	correct: 1,
+	},
+	{
+	question: "What is the correct spelling of this landlocked Asian nation?",
+	choices: ["Kyrgyzstan", "Kurjistan", "Kergystan", "Kyigyzstan"],
+	correct: 0,
+	},
+	{
+	question: "What country is home to the tallest mountain in the world, Mount Everest?",
+	choices: ["Pakistan", "India", "Bhutan", "Nepal"],
+	correct: 3,
+}];
 
-// Compare user's answer and the correct answer to determine score
+var quiz = {
+	questions: questions,
```

Comment:
You had `questions: question` without an 's'. Simple typo here was causing a reference error.